doc/rst/developer/bestPractices/StandardModuleStyle.rst

```diff
@@ -1,3 +1,5 @@
+.. _best-practices-standard-module-style:
+
```

Comment:
Note that line 60 in this file doesn't render right in the generated docs.  Both code snippets need two ` instead of just one